manager/controllers/app/multicluster/multi_cluster_manager.go

```diff
@@ -0,0 +1,75 @@
+package multicluster
+
+import (
+	"fmt"
+	"github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+type ClusterManager interface {
+	GetClusters() ([]Cluster, error)
```

Comment:
I thought that you suggested having a separate interface for GetClusters(). I don't mind if we don't split but just asking if that's really what you decided eventually. 

```go
type ClusterLister interface {
  GetClusters() ([]Cluster, error)
}
type ClusterManager interface {
  ClusterLister 
  GetBlueprint(cluster string, namespace string, name string) (*v1alpha1.Blueprint, error)
  ...
}